src/Agent/NewRelic/Agent/Core/Core.csproj

```diff
@@ -1,145 +1,188 @@
-        <DebugType>full</DebugType>
-        <DebugSymbols>True</DebugSymbols>
-    </PropertyGroup>
-    <ItemGroup>
-        <PackageReference Include="JetBrains.Annotations" Version="10.4.0" />
-        <PackageReference Include="MoreLinq" Version="2.4.0" />
-        <PackageReference Include="log4net" Version="2.0.8" />
-    </ItemGroup>
+﻿<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
-    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">16.0</VisualStudioVersion>
-    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
-    <TransformOnBuild>true</TransformOnBuild>
-    <OverwriteReadOnlyOutputFiles>true</OverwriteReadOnlyOutputFiles>
-    <TransformOutOfDateOnly>false</TransformOutOfDateOnly>
+    <TargetFrameworks>net35;netstandard2.0</TargetFrameworks>
+    <AssemblyName>NewRelic.Agent.Core</AssemblyName>
+    <RootNamespace>NewRelic.Agent.Core</RootNamespace>
+    <Description></Description>
+    <IntermediateOutputPath>$(SolutionDir)\_build\$(Platform)-$(Configuration)\$(AssemblyName)\</IntermediateOutputPath>
+    <OutputPath>$(SolutionDir)\_build\$(Platform)-$(Configuration)\$(AssemblyName)\</OutputPath>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <AssemblyOriginatorKeyFile>$(SolutionDir)NewRelicStrongNameKey.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
   </PropertyGroup>
 
-  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
-  <Import Project="$(VSToolsPath)\TextTemplating\Microsoft.TextTemplating.targets" />
-
-  <PropertyGroup>
-    <ConfigurationCsPath>Config\Configuration.cs</ConfigurationCsPath>
-    <ExtensionCsPath>NewRelic.Agent.Core.Extension\Extension.cs</ExtensionCsPath>
-  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="JetBrains.Annotations" Version="10.4.0" />
+    <PackageReference Include="MoreLinq" Version="2.4.0" />
+    <PackageReference Include="log4net" Version="2.0.8" />
+    <PackageReference Include="ILRepack" Version="2.0.16" />
 
-  <Target Name="GenerateCode" BeforeTargets="CreateGitVersion" Condition="!Exists('$(ConfigurationCsPath)') Or !Exists('$(ExtensionCsPath)')">
-    <PropertyGroup>
-      <Xsd2CodePath>$(SolutionDir)BuildTools\Xsd2Code.exe</Xsd2CodePath>
-    </PropertyGroup>
-    <!-- <Exec Command="&quot;$(Xsd2CodePath)&quot; &quot;$(SolutionDir)NewRelic\Agent\Core\Config\Configuration.xsd&quot; NewRelic.Agent.Core.Config Configuration.cs /cl /ap /sc /xa" /> -->
-    <!-- <Exec Command="&quot;$(Xsd2CodePath)&quot; &quot;$(SolutionDir)NewRelic\Agent\Core\NewRelic.Agent.Core.Extension\Extension.xsd&quot; NewRelic.Agent.Core.Extension Extension.cs /cl /ap /sc /xa" /> -->
-  </Target>
+    <Reference Include="ICSharpCode.SharpZipLib">
+      <HintPath>$(SolutionDir)ExternalLibs\ICSharpCode.SharpZipLib.dll</HintPath>
```

Comment:
ExternalLibs are in src\Agent

Reply:
This is correct, until the FullAgent.sln is moved.

Reply:
understood, i was confused.